src/main/java/attendance/model/Attendance.java

```diff
@@ -0,0 +1,51 @@
+package attendance.model;
+
+import java.time.DayOfWeek;
+import java.time.LocalDateTime;
+import java.time.Month;
+import java.util.Objects;
+
+public class Attendance {
+
+    private final Crew crew;
+    private final LocalDateTime dateTime;
+
+    public Attendance(Crew crew, LocalDateTime dateTime) {
+        validateDateTime(dateTime);
+        this.crew = crew;
+        this.dateTime = dateTime;
+    }
+
+    private void validateDateTime(LocalDateTime dateTime) {
+        boolean isWeekend = dateTime.getDayOfWeek() == DayOfWeek.SATURDAY
+                || dateTime.getDayOfWeek() == DayOfWeek.SUNDAY;
```

Comment:
마음 같아서는 `LocalDateTime` 객체에게 `isHoliday()` 라고 메시지만 보내면 좋을텐데, LocalDateTime은 서프가 구현한 코드가 아니다보니 마음대로 메서드를 추가할 수 있는 상황이 아니네요. 🥲

이런경우에는 `Util` 함수를 만들어볼 수도 있을 것 같아요. 😃 AttendanceTimeline.isCloseDay 메서드에서도 유사한 케이스가 있네요. 👀 

```suggestion
        boolean isWeekend = DateUtils.isHoliday(dateTime);
```

Reply:
감사합니다! 해당부분 반영할게요!